Mail/Views/Thread/MessageBodyView.swift

```diff
@@ -85,8 +75,14 @@ struct MessageBodyView: View {
 
     private func loadBody(blockRemoteContent: Bool) {
         Task {
+            var messageBody = model.showBlockQuote ? presentableBody.body?.value : presentableBody.compactBody
```

Comment:
Maybe we can move all the logic of loadBody to the model
Because before this PR, loadBody was just calling a method from the viewModel